runtime/fundamentals/workspaces.md

```diff
@@ -234,6 +234,100 @@ folowing entries:
 - `@scope/my-package/foo`
 - `@scope/my-package/other`
 
+### Publishing workspace packages to registries
+
+Workspaces make it easy to publish packages to registries like JSR or NPM. You
+can publish individual workspace members while keeping their development
+connected in your monorepo.
+
+#### Publishing to JSR
+
+To publish a workspace package to JSR, follow these steps:
+
+1. Ensure each package has the appropriate metadata in its `deno.json`:
+
+```json title="my-package/deno.json"
+{
+  "name": "@scope/my-package",
+  "version": "1.0.0",
+  "exports": "./mod.ts",
+  "publish": {
+    "exclude": ["tests/", "*.test.ts", "examples/"]
+  }
+}
+```
+
+2. Navigate to the specific package directory and publish:
+
+```sh
+cd my-package
+deno publish
+```
+
+You can also publish directly from the workspace root by specifying the package
+path:
+
+```sh
+deno publish ./my-package
+```
+
+If your workspace contains multiple packages that need to be published together
+with proper versioning, consider using the `--dry-run` flag first to verify what
+files will be included:
+
+```sh
+deno publish ./my-package --dry-run
+```
+
+#### Publishing to NPM
```

Comment:
We don't currently support publishing workspaces to npm. Maybe let's remove this section for now?